napari/_qt/qt_event_loop.py

```diff
@@ -230,6 +253,15 @@ def _(event):
 
         register_threadworker_processors()
 
+        notification_manager.notification_ready.connect(
+            NapariQtNotification.show_notification
+        )
+        notification_manager.notification_ready.connect(
+            show_console_notification
+        )
+
+        app.focusChanged.connect(_focus_changed)
```

Comment:
How come we've moved this connections lower? Is it because we need `get_app` to run? The comment above on line 238 says `running get_app for the first time` but I don't see that call in that `if` block so just trying to understand the flow.

Reply:
No. We should connect notification events only once per process run. Previous code do this on every `get_qapp` call.